util/install/install.Ubuntu.sh

```diff
@@ -1,53 +1,40 @@
 #!/usr/bin/env bash
-# DeepDive installers for Ubuntu Linux
+# DeepDive installers for Debian/Ubuntu Linux
 
-LSB=$(lsb_release -r 2>/dev/null)
-case ${LSB##*	} in
-    12.04|14.04|15.04) true ;;
-    *) error "Ubuntu $LSB found: This installer only works with Ubuntu 12.04, 14.04, and 15.04."
+LSB=$(lsb_release -ir | cut -f2)
```

Comment:
On raiders4 and raiders5 (Ubuntu 12.04), I got `No LSB modules are available` when trying `lsb_release`. Should we deal with `lsb_release` before using it?